invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/ShareModal.js

```diff
@@ -0,0 +1,66 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2021 Northwestern University.
+// Copyright (C) 2023 TU Wien.
+//
+// Invenio RDM Records is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React from "react";
+import { Icon, Button, Modal, Tab } from "semantic-ui-react";
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import PropTypes from "prop-types";
+import { LinksTab } from "./AccessLinks/LinksTab";
+// import { GrantsTab } from "./GrantsTab";
+
+export const ShareModal = ({ record, open, handleClose, accessLinksSearchConfig }) => {
+  const panes = [
+    {
+      menuItem: { icon: "linkify", content: "Links" },
+      pane: (
+        <Tab.Pane className="borderless" attached="top" key="accessLinks">
+          <LinksTab record={record} accessLinksSearchConfig={accessLinksSearchConfig} />
+        </Tab.Pane>
+      ),
+    },
+  ];
+
+  return (
+    <Modal
```

Comment:
**A11y**
The modal trigger button should have
```
aria-expanded={open}
aria-haspopup="dialog"
aria-controls={modalId},
```
and the modal
```
role="dialog"
id={modalId}
```
The first interactive element inside the modal should get focus when opening. (If I understand correctly, you have already implemented this, just make sure to test that it works as expected when using the keyboard.)